app/components/navs/nav-bar.tsx

```
@@ -0,0 +1,51 @@
+
+import React from 'react';
+import TextButton from '../buttons/text-button';
+import FilledButton from '../buttons/filled-button';
+import SearchBox from '../inputs/search-box';
+import { useRouter } from 'next/navigation';
+// import ToggleIconButton from './toggle-icon-button';
+
+export default function NavBar() {
+    const route = useRouter();
+    return (
+        <div className="nav-bar flex flex-grow-0 pl-4 md:pl-0 pt-3 items-center justify-between w-full top-0 max-h-24 sticky bg-inherit z-[1000]" autoFocus={true}>
```

Biome

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)